MobiFlight/MobiFlightModule.cs

```diff
@@ -327,7 +333,10 @@ public void LoadConfig()
                         device.Name = GenerateUniqueDeviceName(buttons.Keys.ToArray(), device.Name);
                         buttons.Add(device.Name, new MobiFlightButton() { Name = device.Name });
                         break;
-
+                    case DeviceType.InputShiftRegister:
+                        device.Name = GenerateUniqueDeviceName(outputs.Keys.ToArray(), device.Name);

```

Comment:
In this line, `GenerateUniqueDeviceName(outputs.Keys...` should be `GenerateUniqueDeviceName(inputShiftRegisters.Keys...`

Reply:
Good catch, this was changed by Doc and I botched a merge. Fixed!